src/pages/Contributing/Designers/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Definition of Done for symbols'
+order:  13
+---
+
+> Mozaic is a worldwide source of truth when it comes to design for Adeo and Leroy Merlin's products. That is why you need to use general guidelines when you contribute to Mozaic. We made some rules to follow when designing new symbols.
+
+This **Definition of Done** page has been divided in 10 steps.
+
+## 1. The pattern doesn't exist in Mozaic
+I checked that the pattern is not in another library :
+- I looked over Mozaic’s website (Foundations / Components)
+- I checked on GitHub (Pull requests)
+- I checked on Jira (Issues in the kanban board)
+
+## 2. The pattern is stored in the right file
+I checked that my pattern is in the right Sketch library and it’s in somewhere that makes sense.
+
+## 3. The naming conventions have been respected in my file
+A design system needs to respond to a certain number of criterias. Naming conventions is one of the most important.
```

Comment:
link to doc

Reply:
will do
but we need to write those : https://mozaic.adeo.cloud/GetStarted/Designers/namingConvention/

Reply:
@tiagoux Indeed ! The naming convention is the next step.
We have already the content, but it's necessary to clean it, and after we can add it to Mozaic 🙃

Reply:
A little purposal to complete the sentence :
"A design system needs to respond to a certain number of criterias. Naming conventions is one of the most important. This helps to have a better files organization files and generates a common language"
What do you think ?